tests/unit/ReportUtilsTest.ts

```diff
@@ -7475,4 +7476,81 @@ describe('ReportUtils', () => {
             expect(actorAccountID).toEqual(123);
         });
     });
+
+    describe('hasEmptyReportsForPolicy', () => {
+        const policyID = 'workspace-001';
+        const otherPolicyID = 'workspace-002';
+        const accountID = 987654;
+        const otherAccountID = 123456;
+
+        const buildReport = (overrides: Partial<Report> = {}): Report => ({
+            reportID: overrides.reportID ?? 'report-1',
+            policyID,
+            ownerAccountID: accountID,
+            type: CONST.REPORT.TYPE.EXPENSE,
+            stateNum: CONST.REPORT.STATE_NUM.OPEN,
+            statusNum: CONST.REPORT.STATUS_NUM.OPEN,
+            total: 0,
+            nonReimbursableTotal: 0,
+            pendingAction: null,
+            errors: undefined,
+            ...overrides,
+        });
+
+        const toCollection = (...reports: Report[]): OnyxCollection<Report> =>
+            reports.reduce<Record<string, Report>>((acc, report, index) => {
+                acc[report.reportID ?? String(index)] = report;
+                return acc;
+            }, {});
+
+        it('returns false when policyID is missing or accountID invalid', () => {
+            const reports = toCollection(buildReport());
+
+            expect(hasEmptyReportsForPolicy(reports, undefined, accountID)).toBe(false);
+            expect(hasEmptyReportsForPolicy(reports, policyID, Number.NaN)).toBe(false);
```

Comment:
Let's update the test here to cover the case when `accountID` equals `CONST.DEFAULT_NUMBER_ID`